Antidote/ChatViewController.m

```diff
@@ -321,10 +321,15 @@ - (void)updateFriendRelatedInformation
 - (void)createPhoneCallBarButton
 {
     UIImage *phoneImage = [UIImage imageNamed:@"call-phone"];
+
     self.navigationItem.rightBarButtonItem = [[UIBarButtonItem alloc] initWithImage:phoneImage
                                                                               style:UIBarButtonItemStylePlain
                                                                              target:self
                                                                              action:@selector(startCallButtonPressed)];
+
+    if (self.friend.connectionStatus == OCTToxConnectionStatusNone) {
+        self.navigationItem.rightBarButtonItem.enabled = NO;
+    }
```

Comment:
Instead of this it would be better to move `createPhoneCallBarButton` before `updateFriendRelatedInformation` in `viewDidLoad` method.